### ISSUE TITLE
Fix `selector-type-no-unknown` false positive for `model`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7692]
### What minimal example or steps are needed to reproduce the bug?

```css
model {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-type-no-unknown": true
  }
}
```

### References

https://developer.apple.com/videos/play/wwdc2023/10279/ @ 13:25
https://immersive-web.github.io/model-element/

### Do you have a proposal to fix the bug?

add to https://github.com/stylelint/stylelint/blob/main/lib/reference/selectors.mjs#L39
